docs/section-3/version-control2.md

```diff
@@ -10,12 +10,12 @@ When you are done reading, let's practice using the Github flow a bit.
 In the pull request, people can add comments, review the code, and merge the code into our `master` branch. This is how we make sure all the code that we put on our robot works and meets our quality standards. We won't be merging your code in this time around, but let a programming lead or mentor know once you get to this step so we can check what you've learned.
 
 ## Good Git Practices
-- Delete branches which are no longer relevant, or do not contain unique code.
+- Delete branches which are no longer relevant or do not contain unique code.
 
 - Add details into your commit message.
 
 A commit called `"Update Motors"` does not help anyone. Give it a title of what you did, and use the message space to actually go into detail about the changes (if needed).
 
-- Use [Github Desktop](https://desktop.github.com/), or, alternatively, github from the command line for comfortable programmers.
+- Use [GitHub Desktop](https://desktop.github.com/), or, alternatively, git from the command line for comfortable programmers.
```

Comment:
Ideally we should still promote using VSCode's "Git" window because it has all the features we need and its annoying to have to download another application, set up all the stuff, etc.